modules/apps/batch-planner/batch-planner-test/src/testFunctional/tests/ImportAndExportObjects.testcase

```diff
@@ -249,14 +286,6 @@ definition {
 
 	@priority = 3
 	test ExportCustomObjectWithoutObjectEntry {
```

Comment:
Why are you deleting this task? I think we need this task, the error show we don't have any object named student 'C_Student (v1_0 - Liferay Object REST)'

Reply:
I don't know how I ended up deleting this task, I believe it was when I went to solve the conflict problem, but I already fixed it